packages/postgrest/lib/src/postgrest_filter_builder.dart

```diff
@@ -478,8 +480,10 @@ class PostgrestFilterBuilder<T> extends PostgrestTransformBuilder<T> {
   /// });
   /// ```
   PostgrestFilterBuilder<T> match(Map query) {
+    assert(!query.values.contains(null),
+        '`null` equality does not work with match. Use isFilter instead.');
     var url = _url;
-    query.forEach((k, v) => url = appendSearchParams('$k', 'eq.$v', url));
+    query.forEach((k, v) => url = appendSearchParams(k, 'eq.$v', url));
```

Comment:
Might be good to undo this.
```suggestion
    query.forEach((k, v) => url = appendSearchParams('$k', 'eq.$v', url));
```